examples/material-ui-remix-ts/package.json

```
@@ -13,7 +14,7 @@
     "@emotion/react": "latest",
     "@emotion/server": "latest",
     "@emotion/styled": "latest",
-    "@mui/material": "next",
+    "@mui/material": "latest",
```

```suggestion
    "@mui/material": "next",
```

For the `master` branch, use `next` for active development. For `v6.x` when this PR is cherry-picked, use the stable version with `latest`.